datamade/la-metro-councilmatic
# Add recently passed meetings to homepage

## Overview

Adds a section to the homepage that displays meetings that have occurred in the past two weeks within the current month. Also adds two unit tests.

### Checklist


### Demo

<img width="882" alt="Screen Shot 2022-03-02 at 3 58 57 PM" src="https://user-images.githubusercontent.com/36973363/156448663-31a66871-c6d4-47b6-82d4-4bf7d9930245.png">


## Testing Instructions

 * Verify that the new section appears and works correctly.
 * Run the tests.

Handles #790
